src/System.Diagnostics.Process/src/System/Diagnostics/Process.OSX.cs

```diff
@@ -186,7 +186,7 @@ private static string GetExePath()
 
         // The ri_proc_start_abstime needs to be converted to milliseconds to determine
         // the actual start time of the process.
-        private const ulong MillisecondFactor = 100000000000;
+        private const int MillisecondFactor = 1000000;
```

Comment:
Should this be called NanosecondsToMillisecondsFactor or something a bit more specific?